test/extended/testdata/bindata.go

```diff
@@ -13606,7 +13613,8 @@ var _examplesDbTemplatesPostgresqlPersistentTemplateJson = []byte(`{
   },
   "message": "The following service(s) have been created in your project: ${DATABASE_SERVICE_NAME}.\n\n       Username: ${POSTGRESQL_USER}\n       Password: ${POSTGRESQL_PASSWORD}\n  Database Name: ${POSTGRESQL_DATABASE}\n Connection URL: postgresql://${DATABASE_SERVICE_NAME}:5432/\n\nFor more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/blob/master/9.5.",
   "labels": {
-    "template": "postgresql-persistent-template"
+    "template": "postgresql-persistent-template",
+    "app": "postgresql-persistent-template"
```

Comment:
The `app` label should be parameterized so that when templates are used in conjunction to form a single application we can make the `app` labels match.